content/advanced/fine-grained-reactivity.mdx

```diff
@@ -0,0 +1,248 @@
+# Understanding Fine-Grained Reactivity
+
+Reactivity in programming establishes automatic responses to data changes, eliminating the need for manual updates to the user interface (UI). 
+By connecting UI elements to the underlying data, updates become automated.
+
+The term "fine-grained reactivity" refers to systems that allow for highly targeted behavioral changes, in contrast to approaches that only support broader, and sometimes unnecessary, updates.
+
+**Note:** If you're new to the concept of reactivity and want to learn the basics, consider starting with our [intro to reactivity guide](/essentials/intro-to-reactivity).
+
+## Reactive Primitives
+
+In Solid's reactivity system, there are two key elements: signals and effects. These core elements serve as the foundation for more specialized reactive features:
```

Comment:
I'd say `signals` and `observers` really. Like memos are pretty core and worth mentioning in one way or another, probably.